tasks/17.txt

```diff
@@ -1,3 +1,15 @@
 Create SQL query to display the average price of each categories's products:
 
-| CategoryName | AvgPrice |
\ No newline at end of file
+| CategoryName | AvgPrice |
+
+db.products.aggregate([
+    {$lookup: {
+        from: "categories",
+              localField: "CategoryID",
+              foreignField: "CategoryID",
+              as: "Categories"
+        }},
+        {$unwind: "$Categories"},
+    {$group: {_id: "$Categories.CategoryName", "AvgPrice": {$sum: "$UnitPrice"}}},
+    {$project: {_id: 0, CategoryName: "$_id", AvgPrice: 1}}
+]);
```

Comment:
We need an "average price of each categories's products".  It is sum of product prices in category now

Reply:
Can be done with one $lookup and one $project